editor/src/components/editor/actions/action-utils.ts

```diff
@@ -131,7 +132,6 @@ export function isTransientAction(action: EditorAction): boolean {
     case 'UNWRAP_LAYOUTABLE':
     case 'UPDATE_JSX_ELEMENT_NAME':
     case 'SET_ASPECT_RATIO_LOCK':
-    case 'SAVE_CURSOR_POSITION':
```

Comment:
oops?

Reply:
I can see why we want to store the cursor position when saving, but having _every single_ cursor change trigger a save was nightmare fuel.